### ISSUE TITLE
Twenty-Sided Die and Evade Builds

### ISSUE THREAD

**Issue by Darwe-Canine** [lyrgard/ffbeEquip#283]
The builder incorrectly tries to use two Twenty-Sided Die accessories when building for evasion (with other criteria present); TSD evasion doesn't stack (which the builder correctly accounts for), but it doesn't select another evade materia instead. This is easily reproducible by building using all equipment and barring Ring of the Lucii and Protec Grappler G3.

**Comment by tomurderthelight** [comment]
makes me wonder what happened to the TSD being marked as unstackable to correct this, since the other aspect of the die isn't/wasn't supported. lost in a data update, probably.

**Comment by lyrgard** [comment]
I added a handling of unstackable abilities. That's why you won't get +40% physical evasion when adding 2 TSD, and why I removed the temporary "unstackable" flag I put on the whole item.

However, I still need to correctly manage the fact that a second TSD is worse than any other option, for evasion. The builder think it is the best option (when 5* TMR are excluded), and see that it can add 2 of them, so it ignore the other source of evasion on accessories. I need the builder to consider only one TSD here, and it will naturally consider the other options.
